tests/Unit/EncodingTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Smalot\PdfParser\Unit;
+
+use Smalot\PdfParser\Document;
+use Smalot\PdfParser\Encoding;
+use Tests\Smalot\PdfParser\TestCase;
+
+/**
+ * @coversDefaultClass \Smalot\PdfParser\Encoding
+ */
+class EncodingTest extends TestCase
+{
+    /**
+     * @covers ::translateChar
+     */
+    public function testTranslateCharOnUnknownGlyph(): void
+    {
+        $encoding = new Encoding(new Document());
+
+        static::assertNull($encoding->translateChar('foo'));
```

Comment:
Assert methods in phpunit have been static since its creation and static code analysis tools like phpstan for example will complain when calling static method dynamically, but both ways are correct. Will change it to dynamic calls to be consistent with the rest of the project.

Reply:
Thank you for the hint though! I was confused about that fact. In the future it totally makes sense to transition to `static` IMHO.